Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:9
```

Comment:
I would put all these files related to dockerising into a docker/ sub folder

Reply:
Docker does not like very much when you reference files outside of context (parent directory of Dockerfile), although things are better in v18. I'll leave it as it is for now.